src/seer_server.erl

```diff
@@ -0,0 +1,49 @@
+-module(seer_server).
+
+-include("seer.hrl").
+
+-define(SERVER, ?MODULE).
+
+-behaviour(gen_server).
+
+-export(
+    [
+        init/1,
+        handle_cast/2,
+        handle_call/3,
+        handle_info/2,
+        terminate/2,
+        code_change/3
+    ]
+).
+-export([start_link/0, stop/0]).
+
+-record(state, {mode :: undefined | stdout}).
+
+start_link() -> gen_server:start_link({local, ?SERVER}, ?MODULE, [], []).
+
+stop() -> gen_server:cast(?SERVER, stop).
```

Comment:
Would you mind clarifying why this exists? Normally processes are terminated by their supervisors.

Reply:
That was copied from something else without a lot of thinking. Removed.